superset/assets/visualizations/deckgl/layers/index.js

```diff
@@ -13,7 +12,6 @@ const layerGenerators = {
   deck_screengrid,
   deck_path,
   deck_hex,
-  deck_scatter,
```

Comment:
removing this here may break the `deck_multi` charts that combine `deck_scatter`